test/SlowTests/Server/Documents/SqlMigrationTests.cs

```diff
@@ -0,0 +1,416 @@
+            }
+        });
+
+        public void Initialize()
+        {
+            var assembly = Assembly.GetExecutingAssembly();
+
+            using (var con = new SqlConnection())
+            {
+                con.ConnectionString = MasterDatabaseConnection.Value;
+                con.Open();
+
+                using (var dbCommand = con.CreateCommand())
+                {
+                    dbCommand.CommandText = string.Format(CreateDatabaseQuery, DatabaseName);
+                    dbCommand.ExecuteNonQuery();
+                }
+            }
+
+            using (var con = new SqlConnection())
+            {
+                con.ConnectionString = GetConnectionString();
+                con.Open();
+
+                using (var dbCommand = con.CreateCommand())
+                {
+                    var textStreamReader = new StreamReader(assembly.GetManifestResourceStream("SlowTests.Data.create.txt"));
+                    dbCommand.CommandText = textStreamReader.ReadToEnd();
+                    dbCommand.ExecuteNonQuery();
+                }
+
+                using (var dbCommand = con.CreateCommand())
+                {
+                    var textStreamReader = new StreamReader(assembly.GetManifestResourceStream("SlowTests.Data.insert.txt"));
+                    dbCommand.CommandText = textStreamReader.ReadToEnd();
+                    dbCommand.ExecuteNonQuery();
+                }
+            }
+        }
+
+        public static string GetConnectionString()
+        {
+            return MasterDatabaseConnection.Value + $";Initial Catalog={DatabaseName};";
+        }
+
+        [Fact]
+        public void CanMigrateSqlDatabase()
+        {
+            using (var store = GetDocumentStore())
+            {
+                Initialize();
+
+                var tablesToWrite = new List<SqlMigrationTable>
+                {
+                    new SqlMigrationTable("dbo.Customer")
+                    {
+                        EmbeddedTables = new List<SqlMigrationTable>
+                        {
+                            new SqlMigrationTable("dbo.Order")
+                            {
+                                EmbeddedTables = new List<SqlMigrationTable>
+                                {
+                                    new SqlMigrationTable("dbo.OrderItem")
+                                    {
+                                        EmbeddedTables = new List<SqlMigrationTable>
+                                        {
+                                            new SqlMigrationTable("dbo.Details")
+                                        }
+                                    }
+                                }
+                            },
+                            new SqlMigrationTable("dbo.Photo")
+                        }
+                    }
+                };
+                var operation = new SqlMigrationOperation(GetConnectionString(), true, true, true, false, tablesToWrite);
```

Comment:
use named parameters for the booleans to make it easier to figure out what you are doing